### PR TITLE
Remove wizAssets dependency

### PR DESCRIPTION
**Pull request by cstoquer**
Removed `wizAssets` dependency from `AudioManager`.
Instead, a custom loader function can be specified in `settings.getFileUri`

Add the following code to use `wizAssets` as loader:
```javascript
if (window.wizAssets) {
	audioManager.settings.getFileUri = function wizAssetsGetUri(audioPath, id, cb) {
		window.wizAssets.downloadFile(audioPath + id + '.mp3', 'audio/' + id + '.mp3', 
			function onSuccess(uri) {
				cb(null, uri)
			}, 
			cb // onFail callback
		);
	}
}
```

### CODE REVIEW

File: Sound.js [hunk 1]
```diff
@@ -72,10 +70,24 @@ Sound.prototype._load = function (filePath) {
...
+		getFileUri(audioPath, this.id, function onUri(error, uri) {
+			if (error) return loadFail(error);
+			loadAudio(uri);
+		});
 	} else {
-		loadAudio(filePath);
+		// synchronous
+		try {
+			var uri = getFileUri(audioPath, this.id);
+			// TODO: check uri existence
```
Comment (ronkorving) [inline Sound.js]: How?

File: SoundBuffered.js [hunk 2]
```diff
@@ -192,14 +190,29 @@ SoundBuffered.prototype._load = function (filePath) {
...
+		getFileUri(audioPath, this.id, function onUri(error, uri) {
+			if (error) return loadFail(error);
+			loadAudio(uri);
+		});
 	} else {
-		loadAudio(filePath);
+		// synchronous
+		try {
+			var uri = getFileUri(audioPath, this.id);
+			// TODO: check uri existence
```
Comment (ronkorving) [inline SoundBuffered.js]: See above.

File: Sound.js (hunk 1 above)
Comment (cstoquer) [inline Sound.js]: My intention was only to check that result `uri` is not empty.
I'll just add the code, no need to leave this TODO.

**Comment by cstoquer** [comment]
updated

**Comment by ronkorving** [comment]
I would've totally dropped the check, but you gotta do what you gotta do :) Go for it and merge this please.

**Pull request merged**